### PR TITLE
New Feature: Storage and Syncing progress

### PR DESCRIPTION
**Pull request by AB261**
Users can now view the number of emails stored and the progress of syncing of those emails on the server page. This can also be used to check how many emails were synced in comparison to stored. 
UI needs to be made better still. 

Fixes issue #38 
Fixes issue #34

### CODE REVIEW

**Comment by jupancs** [comment]
The pr works. It seems like the progress bar refelect the progress of email synchronization between the phone and the laptop? From your commit, it looks like you want to refelect the progress of email storing, and I think the storing progress is actually what we want.

**Comment by jupancs** [comment]
As we know the number of downloaded emails, can we also add a progress bar for the "stored email"? Or a text status like "Stored emails: 2 / 4"

**Comment by jupancs** [comment]
The synchronizing progress bar finished earlier than the actual synchronization finished.

File: mailsync-anroid/app/src/main/java/edu/ua/cs/nrl/mailsync/EmailRepository.java [hunk 1]
```diff
@@ -72,6 +75,34 @@ public EmailRepository(){
...
+        incompleteUids.add(uid);
+        isIncomplete=true;
+    }
+    //Notifies user if an email is not stored completely
+    public void notifyIncompleteEmail(long uid){
+        toast(context,"The email of " + uid + " was not stored correctly. Please try again later");
+    }
+
+    //Returns a boolean that returns true if there are any incomplete emails
+    public static boolean isIsIncomplete() {
```
Comment (jupancs) [inline mailsync-anroid/app/src/main/java/edu/ua/cs/nrl/mailsync/EmailRepository.java]: Is there any typo in the method name "isIsIncomplete()"? Should it be "ifIsIncomplete()" or just "isIncomplete"?

Comment (AB261) [reply mailsync-anroid/app/src/main/java/edu/ua/cs/nrl/mailsync/EmailRepository.java]: Yeah let me make that more clear.. I used android automatic function creator which makes a function for the boolean variable giving it the name isbooleanname()

**Pull request merged**